src/include/platform/internal/GenericPlatformManagerImpl_FreeRTOS.cpp

```diff
@@ -241,6 +241,21 @@ void GenericPlatformManagerImpl_FreeRTOS<ImplClass>::PostEventFromISR(const Chip
 template <class ImplClass>
 CHIP_ERROR GenericPlatformManagerImpl_FreeRTOS<ImplClass>::_Shutdown(void)
 {
+    VerifyOrDieWithMsg(false, DeviceLayer, "Shutdown is not implemented");
```

Comment:
Why assert here? And why is shutdown not implemented on FreeRTOS? Is there an issue tracking this?

Reply:
I'm asserting because in general, during shutdown, folks rarely check the return values of these calls (this should really be 'void' perhaps?), and since it's not implemented, the contractual expectation of this API is not held and should result in an assert to make it easier to discover this later when it gets thrown on embedded platforms.